### ISSUE TITLE
Odd message printed by 'grunt modulify'

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/chipper#1328]
When I run `grunt modulify`, I see a new message:

```
% cd beers-law-lab
% grunt modulify
Running "modulify" task
modulifying beers-law-lab
Found JS string file in TS repo.  It should be deleted manually.  ../beers-law-lab/js/BeersLawLabStrings.ts
info: git add js/BeersLawLabStrings.ts on beers-law-lab
```

I see 2 JS files in this repo: Gruntfile.js and beers-law-lab-phet-io-overrides.js, neither of which should be deleted or converted to TS.

What is a "TS repo"?

And what is the point? I can't tell from the commit message where this was introduced: 1fd0f9207c04e0c4e90856750846d3b047578c2c

**Comment by samreid** [comment]
It looks like this was introduced in https://github.com/phetsims/chipper/commit/13f5fa5a71e52ba5563a365da28c760bc70901a7 during https://github.com/phetsims/simula-rasa/issues/26.  @chrisklus and/or @marlitas can you please take a look?

**Comment by marlitas** [comment]
From my understanding this error is claiming that there is a javascript Strings file in a repo that has been converted to typescript (and should therefore be a typescript file).

This error message was triggered by a change @chrisklus and I introduced in one of the string comparisons in `Modulify.ts`. We accidentally changed a `.js` suffix to a `.ts` suffix. I reversed that and all should be working well again. 

Sending back to @pixelzoom to double check on his end.
